src/compiler/transformers/esnext.ts

```diff
@@ -10,6 +10,23 @@ namespace ts {
         ClassAliases = 1 << 1,
     }
 
+    /**
+     * A mapping of private names to information needed for transformation.
+     */
+    type PrivateNameEnvironment = UnderscoreEscapedMap<PrivateNamedInstanceField>;

```

Comment:
as discussed, we will fiddle with this structure more in subsequent PRs (so that methods + static methods + getters can share the same WeakSet of instances)